iOS Client/ArticlesController/MBArticleDetailViewController.swift

```diff
@@ -25,24 +25,55 @@ class MBArticleDetailViewController: UIViewController, WKNavigationDelegate {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        configureWebView()
-        configureBackButton()
-    }
-    
-    func configureWebView() {
+        
         let webConfiguration = WKWebViewConfiguration()
         webView = WKWebView(frame: .zero, configuration: webConfiguration)
         webView.navigationDelegate = self
         view = webView
         
+        configureWebView()
+        
+        configureBackButton()
+        
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(contentSizeDidChange(_:)),
+                                               name: NSNotification.Name.UIContentSizeCategoryDidChange,
+                                               object: nil)
+    }
+    
+    @objc private func contentSizeDidChange(_ notification: Notification) {
+        configureWebView()
+    }
+    
+    func configureWebView() {
         if let content = self.selectedArticle?.content {
             print(content)
-            let cssHead: String = "<head>\n<link rel=\"stylesheet\" type=\"text/css\" href=\"MB.css\">\n</head>\n"
+            // <head>
+            let contentTypeHead = "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\">"
+            let viewPortHead = "<meta name=\"viewport\" content=\"width=device-width,initial-scale=1\">"
+            let cssHead = "<link rel=\"stylesheet\" type=\"text/css\" href=\"MB.css\" />"
+            let head = "<head>\n\(contentTypeHead)\n\(viewPortHead)\n\(cssHead)\n</head>"
+            
+            // <style>
+            let titleFontSize = UIFont.preferredFont(forTextStyle: .headline).pointSize
+            let authorFontSize = UIFont.preferredFont(forTextStyle: .subheadline).pointSize
+            let bodyFontSize = UIFont.preferredFont(forTextStyle: .body).pointSize
+            let style = ""
```

Comment:
The only way to adjust the size in the web view is by giving it a new css file? I thought there would be some properties on the webview but if not then we can stick with this

Reply:
yeah . . . I was hoping it would be built-in for the wk web view but that wasn't the case . . . which led to this.